model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -919,15 +922,21 @@ async def execute(
                             token=result["result"]["token"]["text"],
                             log_prob=result["result"]["token"]["logprob"],
                         )
-                    yield CompletionStreamV1Response(
-                        request_id=request_id,
-                        output=CompletionStreamOutput(
-                            text=result["result"]["token"]["text"],
-                            finished=finished,
-                            num_completion_tokens=num_completion_tokens,
-                            token=token,
-                        ),
-                    )
+                    try: 
+                        yield CompletionStreamV1Response(
+                            request_id=request_id,
+                            output=CompletionStreamOutput(
+                                text=result["result"]["token"]["text"],
+                                finished=finished,
+                                num_completion_tokens=num_completion_tokens,
+                                token=token,
+                            ),
+                        )
+                    except Exception as e:  # if result["result"]["token"]["text"] doesn't exist 
+                        logger.exception(f"Error parsing text-generation-inference stream output. Error message: {e}")
+                        raise ObjectHasInvalidValueException(
+                            f"Error parsing text-generation-inference stream output." 
```

Comment:
@yunfeng-scale the error that's returned to the user here if I were to parse the error is just a KeyError, which is confusing to return (stems from the fact that result["result"]["token"]["text"] doesn't actually exist), do you think handling it this way is fine?

Reply:
what about just check whether result contains "error" as a key